src/main/scala/com/databricks/spark/xml/util/XmlFile.scala

```diff
@@ -101,6 +101,10 @@ private[xml] object XmlFile {
 
```

Comment:
@pwoody, mind if I ask to remove `val startElement = s"<${options.rootTag}>"` and `val endElement = s"</${options.rootTag}>"` above? Seems unused.

Reply:
Yep, done.